backend/src/http/handlers.rs

```diff
@@ -101,7 +95,15 @@ pub(super) async fn handle(req: Request<Body>, ctx: Arc<Context>) -> Response {
         // duplicate logic. So yeah:
         //
         // TODO: fix that at some point ^
-        _ => ctx.assets.serve_index(StatusCode::OK, &ctx.config).await,
+        _ => {
+            let noindex_prefixes = ["/!", "/~search", "/~manage"];
```

Comment:
Are there any `/~`-pages that we **want** to be indexed? I would be inclined to just exclude those as well. Let's hope for example we won't forget to add `/~embed` here. x)

Reply:
`/~about`, `/~login`, `/~upload` are left. What do you think? I guess it would be cool to index the about page to find all tobira instances via google :D the other ones... well. The upload route will likely never be indexed because it's only available when logged in.

Reply:
I think `/~login` shouldn't be indexed. `/~upload` maybe, if it can be accessed without a login, which I take it it can't, so that's safe to exclude as well.

But if `/~about` is really the only sensible exception here I would rather see a special case for **that**, I think. :thinking: